examples/fullstatus.py

```diff
@@ -2,14 +2,11 @@
 
 from juju.client.connection import Connection
 from juju.client.client import ClientFacade
-
-
-loop = asyncio.get_event_loop()
-conn = loop.run_until_complete(Connection.connect_current())
-
+from juju import loop
 
 async def status():
     client = ClientFacade()
```

Comment:
Yeah, this has been implemented in Model.get_status() since this example was created. But I don't think it hurts to have some examples of calling APIs directly too. Although this might be a bad example to use for that now since there's a way to do it in the object layer.